jfr-jlink/build.gradle.kts

```diff
@@ -53,6 +53,9 @@ tasks.register<Copy>("copySources") {
     description = "Copies sources from other subprojects"
     into(".generated-src")
     from("../jfr-daemon/src/main/java") {
+        //jlink binary will not need customized logging of slf4j. By excluding, it prevents a
+        //split package issue in the binary
+        exclude ("**org**")
```

Comment:
We need a stricter exclude here, so that in the future we don't inadvertently exclude packages that start with org but are unrelated to logging.

Reply:
I updated this with the latest commit. Tested it and it is very specific now.